Java/CryptoDemo/src/main/java/com/breadwallet/cryptodemo/TransferListActivity.java

```diff
@@ -158,6 +161,36 @@ public Void visit(TransferDeletedEvent event) {
         });
     }
 
+    private List<? extends Transfer> getTransfers() {
```

Comment:
Ordering is kind of nasty. Do we want to have one or more comparator functions at the C layer?

Reply:
Yes, I think we need comparator functions in the C level - nominally {block number, block transaction index} w/ DESC (it seems) for confirmed transaction but some timestamp (+ nonce) fallback for unconfirmed transactions.  And then some suitable Java/Swift interface.  Could be a wallet default (a little hesitant to go there but if Swift/Java need to store the default themselves, might as well be in the Core) and an override option?

Reply:
CORE-400 created.